rust-runtime/aws-smithy-runtime/src/client/orchestrator.rs

```diff
@@ -216,7 +216,7 @@ async fn make_an_attempt(
     Ok(checkpoint)
 }
 
-#[cfg(all(test, feature = "test-util"))]
+#[cfg(all(test, feature = "test-util", feature = "anonymous-auth"))]
```

Comment:
I don't think it's just for testing, but I'm not 100% on that yet. The question will be: is the absence of an auth scheme/option automatically anonymous, or should we keep this thing around? Probably the former so that a model can explicitly say that an operation doesn't require auth. However, I don't think Smithy has any way to communicate that currently.